less/styles.less

```diff
@@ -50,6 +50,10 @@
   font-size: 100px;
 }
 
+.font-roboto {
```

Comment:
I don't think we should be introducing a new font here. Why are we not using the website font default?

Reply:
I see the design mockup is using roboto, I thought they want to use it? 🤔

If not, I'm gonna remove

Reply:
I find it probably due to the design tool, it seems automatically use this font... 🤔

Reply:
removed